app/utils/WkConf.scala

```diff
@@ -132,12 +132,9 @@ class WkConf @Inject()(configuration: Configuration, certificateValidationServic
     val publicDemoDatasetUrl: String = get[String]("features.publicDemoDatasetUrl")
     val exportTiffMaxVolumeMVx: Long = get[Long]("features.exportTiffMaxVolumeMVx")
     val exportTiffMaxEdgeLengthVx: Long = get[Long]("features.exportTiffMaxEdgeLengthVx")
-    val openIdConnectEnabled: Boolean =
-      featureOverrides.getOrElse("openIdConnectEnabled", get[Boolean]("features.openIdConnectEnabled"))
-    val editableMappingsEnabled: Boolean =
-      featureOverrides.getOrElse("editableMappingsEnabled", get[Boolean]("features.editableMappingsEnabled"))
-    val segmentAnythingEnabled: Boolean =
-      featureOverrides.getOrElse("segmentAnythingEnabled", get[Boolean]("features.segmentAnythingEnabled"))
+    val openIdConnectEnabled: Boolean = get[Boolean]("features.openIdConnectEnabled")
+    val editableMappingsEnabled: Boolean = get[Boolean]("features.editableMappingsEnabled")
+    val segmentAnythingEnabled: Boolean = get[Boolean]("features.segmentAnythingEnabled")
   }
```

Comment:
@fm3 Sorry to ping you but I need another quick check from your side. I forgot to remove the manual overwrite handling in the `WkConf`'s `Features` object, which is now already done in 18ll of this file. So this is just a small clean-up.

I tested this via println output in the backend to see whether the overwrites were actually applied. So imo these changes should be fine